web/src/components/mainPageCarousel/mainPageCarousel.tsx

```diff
@@ -0,0 +1,26 @@
+import Carousel from '../shared/carousel/Carousel';

```

Comment:
name mainPageCarousel.tsx as MainPageCarousel.tsx since its a component

Reply:
![image](https://user-images.githubusercontent.com/46846078/164012170-9c75113f-8b69-4150-95a6-6be0ceb3b0a8.png)
Changed file name but git didn't catched it